private/buf/bufcheck/internal/bufcheckserver/internal/bufcheckserverutil/bufcheckserverutil.go

```diff
@@ -78,6 +78,22 @@ func NewRuleHandler(
 	)
 }
 
+// NewMultiHandler returns a new check.RuleHandler combined from multiple handlers.
```

Comment:
This is the old [combine function](https://github.com/bufbuild/buf/blob/c11801c352ea5c693faa3b8f43ec886e1339d765/private/bufpkg/bufcheck/buflint/internal/buflintcheck/util.go#L335)